### PR TITLE
Refactor/internal/show build info crashtime kpango refactoring

### PR DESCRIPTION
**Pull request by kpango**
### Description:

refactor of #184 

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [x] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kpango** [comment]
Please wait merge, I'll send small fix about Error

File: internal/info/info.go [hunk 1]
```diff
@@ -19,93 +19,183 @@ package info
...
+			d.GitCommit = "master"
+		}
+		if len(Version) == 0 && len(d.Version) == 0 {
+			d.Version = GitCommit
+		}
+		if len(d.BuildTime) == 0 {
+			d.BuildTime = BuildTime
+		}
+		if len(d.GoVersion) == 0 {
+			d.GoVersion = runtime.Version()
```
Comment (rinx) [inline internal/info/info.go]: :100:

File: internal/runner/runner.go [hunk 2]
```diff
@@ -76,24 +74,30 @@ func Do(ctx context.Context, opts ...Option) error {
...
+		log.Infof("\n%s", info.String())
 		return nil
 	}
 
 	cfg, version, loc, err := r.loadConfig(p.ConfigFilePath())
 	if err != nil {
 		return err
 	}
+
+	info.Init(r.name)
```
Comment (rinx) [inline internal/runner/runner.go]: L77 and L87 could be declare before L76.

**Comment by kpango** [comment]
@rinx fixed

**Pull request merged**